vendor/assets/javascripts/fixedsticky.js

```diff
@@ -0,0 +1,191 @@
+;(function( win, $ ) {
+
+  function featureTest( property, value, noPrefixes ) {
+    // Thanks Modernizr! https://github.com/phistuck/Modernizr/commit/3fb7217f5f8274e2f11fe6cfeda7cfaf9948a1f5
+    var prop = property + ':',
+      el = document.createElement( 'test' ),
+      mStyle = el.style;
+
+    if( !noPrefixes ) {
+      mStyle.cssText = prop + [ '-webkit-', '-moz-', '-ms-', '-o-', '' ].join( value + ';' + prop ) + value + ';';
```

Comment:
Oh I see what it's doing, it's adding vendor prefixes to the property it's working with and appending them to the existing string.  That `';'` is actually separating one property/value combination from the next, but I read it to be doing that between the property and value initially.